coresimd/x86/tbm.rs

```diff
@@ -37,6 +38,7 @@ pub fn _bextr_u32(a: u32, start: u32, len: u32) -> u32 {
 /// the least significant bits of the result.
 #[inline]
 #[target_feature(enable = "tbm")]
+#[stable(feature = "simd_x86", since = "1.27.0")]
```

Comment:
I think it would be better to remove this in this comment. Not that we fix it some day and it suddenly becomes stable.